src/site/topic25.rst

```diff
@@ -18,9 +18,55 @@ Topic #25 --- Sorting
 
 * `Here is a simple visualization of some popular sorting algorithms <https://www.toptal.com/developers/sorting-algorithms>`_
 
+
 Sorting Problem
 ===============
 
+* Given a collection of *things*
+    * Numbers
+    * Strings
+    * People
+    * Car Parts
+
+* The goal is to arrange the collection of *things* such that they are *in order*
+
+
+Order
+-----
+
+* What *in order* means however will depend on what is being sorted
+* For example
+    * Numbers come with an intrinsic ordering
+    * Strings could be ordered alphabetically
+        * Perhaps by length?
+        * Maybe based on the character's ascii values?
+        * Treat the strings as base 26 integers?
```

Comment:
This go a little out there a little too quickly. Stick with simple examples like alphabetically, case-insensitive alphabetical, length. The other objects (people, car parts) already dive into weird sorting properties.